web/src/App.jsx

```
@@ -4376,68 +4353,74 @@ function App() {
+                          className="ghost-btn icon-only"
                           onClick={() => {
                             setContactSearch('');
                             contactSearchRef.current?.focus();
                           }}
-                          style={{ marginLeft: 4, padding: 0, textDecoration: 'underline' }}
+                          aria-label="Clear search"
+                          title="Clear search"
+                          style={{ width: '28px', height: '28px' }}
                         >
-                          Clear search
+                          <CloseIcon />
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)